src/components/fileHandling/DescriptiveStatistics/DSDropLists.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { Component, ReactNode } from "react";
+import { descriptiveStatisticData, listProps, addDetState, listNode } from './index.ts';
+
+
+
+export class DisplayDSList extends Component<listProps, addDetState> {
```

Comment:
please use functional components unless you have a reason to use class components.